### ISSUE TITLE
Cohort Planning Board

### ISSUE THREAD

**Issue by SallyMcGrath** [CodeYourFuture/Programme#64]
Getting ready for Sheffield, our plan is to support the new Programme Delivery team to launch a cohort in the way we launched North West (an improvement on that actually).

### Overall goal

Our goal is to launch a calm, friendly, self-organised community with low drama and high engagement. We want to meet our regulatory requirements in the simplest way possible, and empower our volunteers to run the classes with the trainees in a collaborative and grown up way.

https://github.com/orgs/CodeYourFuture/projects/110/views/1

### Specific task

Please review the (fab! Thanks @Dedekind561!) cohort launch issues and add context, links, and simple guidance based on what we've all learned running NW6 so far. Do not unnecessarily assign tickets to staff -- keep this for regulatory risks (safeguarding, insurance, GDPR). Assign to the milestones I have set up instead of using labels if possible. Overall, let's keep our eye on clarity and simplicity.

### Links, notes, resources

1. Here's the Miro I made mapping this journey from the trainees' perspective https://miro.com/app/board/o9J_kujEdws=/
(This board should be from the community perspective. )

2. I have provided pre-written comms for the launch process as well as template spreadsheets and forms. These are as simple and minimal as possible and stored in https://drive.google.com/drive/folders/1ljfmzX0HW9-OkA5DCNN4c6spgA3nPLvu

3. I have rewritten the [Personal Circumstances form](https://docs.google.com/document/d/10InrZgoyQye0lwa6okL1px8HBnm3CeD32yyZCJOnECA/edit?usp=sharing), [Access Needs](https://forms.gle/ccyLgChNjJhtdRFh9) form, and created [the signposts site](https://signposts.codeyourfuture.io/) to handle pastoral signposting and manage/secure data collection. **We must minimise data collection.** There's still too much happening.

(Instead of linking directly to specific files/docs, link to folders/sites so we can change out the docs without breaking these issues. )

#### Mini deadline

We'll do a session with the Proms on this board in about ten days so please try to take a look before then and make some updates. For example, Mariana you may wish to write a short checklist for Cohort Founders.  This session will be roles and responsibilities. We will do more sessions.

Let's also iterate on our strategy document for launching Sheffield, incorporating @gbencci 's notes and changes. Is agreeing a strategy the first task in launching a cohort? More possible tickets in comments.

**Comment by kfklein15** [comment]
@SallyMcGrath , should this be the milestone rather than 1 ticket - and we'll have smaller tickets? If yes, let me know and I can make this change

**Comment by SallyMcGrath** [comment]
Oh good note, @kfklein15 . Yes please! 🙏

**Comment by SallyMcGrath** [comment]
@Dedekind561 and @MarianaVazquezCYF what is your progress on this?

@kfklein15 I cannot find any place in our calendars for us to meet about this for weeks, so let's not have that call be a blocker, Please run over the tickets and make any additions/deletions. B

I've been adding context on the tickets as much as pos (without linking to specific pages as those links will die, just going to general places like signposts)

At some point I will go and replace the text on the ITD steps with the steps stored as version 4. I definitely suggest reading them as many steps previously done after ITD are now done during (eg Rainbird)

So far as I understand it, we can't switch to version 4 as this would require everyone who has signed up to get a new email address? (This sounds mad but I swear I didn't dream it.)

**Comment by Dedekind561** [comment]
Hey there, 
We did a first pass going through the issues on the board. We did the following:
- Added a **Community setup** milestone with steps that need completing _before_ ITD launch
- In general, we made it so that the steps in each section should block the start of a new section. In other words, you have to finish ITD launch before going on to ITD land etc.
- We transferred some old steps from [CYF Guide launching a cohort](https://docs.google.com/document/d/1vwKuyAslQwWH4DOtfqa20baFili0cupor3LRES6XPgI/edit) and prepared the ground for the doc's deletion

**Comment by Dedekind561** [comment]
The work was captured in this issue: https://github.com/orgs/CodeYourFuture/projects/129/views/1?pane=issue&itemId=53883109
I'm going to de-assign us now so others can pick this up.

**Comment by kfklein15** [comment]
I have created a milestone and added the tickets I could see on our board.
https://github.com/CodeYourFuture/Programme/milestone/9

Should no 1, 2 and 3 of your post be separate tickets in this milestone or would you prefer keeping this ticket, @SallyMcGrath ?

If you prefer keeping this ticket, I suggest amending the title explaining what this is, since it is a bit generic.

**Comment by MarianaVazquezCYF** [comment]
Hello, I am super confused about the comment on slack  (https://codeyourfuture.slack.com/archives/C05KCLBLG48/p1709069868598329?thread_ts=1709064729.794909&cid=C05KCLBLG48) about this specific task! there were a few things mentioned on this ticket, but the actions in my understanding, were: 

- review the work, Assign to the milestones you set up instead of using labels, and update with things we need, for example, a checklist for cohort founders! Therefore this is what we did! 
- Summarise the Sheffield strategy with German comments, (I did another ticket about this) 

Let's have a meeting because I don't know what we did wrong, and of course, we didn't throw all your work (not intentionally)  why should we?  :( 

Therefore, I think this was more a problem with communication and tickets.

**Comment by SallyMcGrath** [comment]
Yes I think it's gone wrong somewhere because I asked a bunch of questions on the tickets which nobody answered, and you made the old process again, instead of looking at the one I had worked on. You added duplicate tickets and tickets that were no longer needed, from the old process. 

Then you opened the board to the staff and they started adding even more tickets of tasks I have eliminated. 

It didn't help me at all and now I have to go and sort out the mess and then do this ticket myself I think? As we're on a deadline and you are away. 

I definitely would like some info on how I could have said this more clearly and what to do next time. Would also like to understand why you didn't respond to the questions on the board.

**Comment by MarianaVazquezCYF** [comment]
Perfect lets talk in our call and I can share my screen about how I see github, I normally go to this link: https://github.com/issues/ and I see the tickets I have been assigned or I have been mentioned, and I don't see any ticket where I need to respond to questions! So maybe I am missing something ? 

and yes I understand the frustration, I am also frustrated, I was on another deadline and I needed to work on this ticket for extra 2 hours ... :( so its challenging so before if passes again, let's see where we all made a mistake.

**Comment by Dedekind561** [comment]
Hey folks, here are my notes on why this was unclear:

- this wasn't an issue, it was a milestone
- the instructions below suggested we needed to update the board:
> Please review the (fab! Thanks @Dedekind561!) cohort launch issues and add context, links, and simple guidance based on what we've all learned running NW6 so far.

And then:
> Assign to the milestones I have set up instead of using labels if possible. Overall, let's keep our eye on clarity and simplicity. 

I have interpreted this as **reviewing and updating** the cohort planning project board. I've further interpreted this as **assign issues to the milestones** - so I thought your intention involved us adding tickets to the milestones you created on the planning board.  So we added some additional tickets to the board against the milestones using the old [Launching a cohort document](https://docs.google.com/document/d/1vwKuyAslQwWH4DOtfqa20baFili0cupor3LRES6XPgI/edit) (which hadn't been deleted).

And also:
> We'll do a session with the Proms on this board in about ten days so please **try to take a look before then and make some updates**.

This isn't very clear because it is in the **Next Steps** section but is also the **Specific Task**. This is part of why I assumed this meant that we were to **review and update** the board.


**In summary**

In my head, the existing cohort planning board was based on this document:
https://docs.google.com/document/d/1vwKuyAslQwWH4DOtfqa20baFili0cupor3LRES6XPgI/edit

So, in my mind, it made sense to double-check this document for any missing steps and add them to the planning board (which is from the community perspective and not the trainee perspective). This was made more confusing as this document hadn't been deleted - so I assumed it still contained useful information.

One problem is that I'm not confident about the source of truth anymore (but I was reasonably confident about it before going into this piece of work). I now have far less confidence in the source of truth.

**Comment by Dedekind561** [comment]
Re questions:

> Is agreeing a strategy the first task in launching a cohort?
 
I didn't answer this question because it was in Next Steps. In my head it made sense to review and update the board and then deal with the actions in the Next steps section.

> So far as I understand it, we can't switch to version 4 as this would require everyone who has signed up to get a new email address?

This sounds like you're unsure of something instead of really asking us about something. I made this assumption because I'm coming to this piece of work cold.

**Comment by AmyCYF** [comment]
@SallyMcGrath can I edit the existing project or would you like me to create a new one so you can retain the original?
